src/utils/error-handle.js

```diff
@@ -0,0 +1,6 @@
+var util = require('util');
+
+module.exports = function handleSystemError(message, ...objs){
```

Comment:
We've discussed this and, for now, this is 'good enough'. Ill correct this in a later PR.